qcodes/utils/installation_info.py

```diff
@@ -77,28 +67,12 @@ def get_qcodes_requirements_versions() -> Dict[str, str]:
 
     req_names = get_qcodes_requirements()
 
-    req_modules = []
-
-    for req_name in req_names:
-        # the requirement might have a pep 496
-        # env marker. Filter that out before
-        # checking the version
-        req_name = req_name.split(';')[0]
-        if req_name in _VERSIONLESS_PACKAGES:
-            pass
-        elif req_name in _IMPORT_NAMES:
-            req_modules.append(_IMPORT_NAMES[req_name])
-        else:
-            req_modules.append(req_name)
-
     req_versions = {}
 
-    for req_module in req_modules:
-        mod = importlib.import_module(req_module)
-        if req_module in _PACKAGE_NAMES:
-            req_pkg = _PACKAGE_NAMES[req_module]
-        else:
-            req_pkg = req_module
-        req_versions.update({req_pkg: mod.__version__})  # type: ignore
+    for req in req_names:
+        try:
+            req_versions[req] = version(req)
```

Comment:
Unlike the previous implementation this captures extra requirements too. These may not be installed so here we skip import errors. Should we be more clever on that?

Reply:
> more clever

i can only think of logging.

Reply:
Ok since the points of this is logging the info I have opted to log this with a version of "None"